.rubocop.yml

```diff
@@ -14,7 +14,7 @@ Metrics/LineLength:
 
 Metrics/MethodLength:
   CountComments: false
-  Max: 26
+  Max: 50
```

Comment:
50 lines really is pretty long for a method. Maybe we can split the offending method apart a bit?